packages/jellyfish-transaction/src/script/dftx/dftx_icx.ts

```diff
@@ -0,0 +1,50 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import BigNumber from 'bignumber.js'
+import { Script } from '../../tx'
+import { CScript } from '../../tx_composer'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+export enum ICXOrderType {
+  /** type for DFI/BTC orders */
+  INTERNAL = 1,
+  /** type for BTC/DFI orders */
+  EXTERNAL = 2
+}
+
+/**
+ * ICX CreateOrder DeFi Transaction
+ */
+
+export interface ICXCreateOrder {
+  orderType: number
```

Comment:
add comments with byte length. would be helpful for the first-time reader. 
https://github.com/DeFiCh/jellyfish/blob/99b0d0fd894b9c2b11c53860061ee59d7bde9ddf/packages/jellyfish-transaction/src/script/dftx/dftx_governance.ts#L10

Reply:
9c5234c